src/hooks/prisma.ts

```diff
@@ -110,11 +117,32 @@ function createPrismaClientMethodProxy<T extends (...args: unknown[]) => unknown
       if (argArray?.length > 0) {
         const requestParams = argArray[0] as PrismaRequestParams;
         if (requestParams.action && requestParams.model) {
-          prismaCall = recording.functionCall(
-            getFunctionInfo(requestParams.model, requestParams.action, moduleId),
-            requestParams.model,
-            [requestParams.args?.data, requestParams.args?.include, requestParams?.args?.where],
-          );
+          // In extended prisma clients, we receive repeated calls
+          // with the same action, module and args. It can be seen
```

Comment:
I suspect the problem might be that we're patching the same client multiple times. You can double check by looking at `this`, `this.prototype` and the proxy; if I'm right, the calls will have the same `this` and `prototype` but different proxies. In this case, perhaps you can track the created proxies using a `WeakSet` to ensure you're not repatching.